cmd/libs/go2idl/client-gen/main.go

```diff
@@ -35,6 +35,8 @@ func main() {
 		"k8s.io/kubernetes/pkg/fields",
 		"k8s.io/kubernetes/pkg/labels",
 		"k8s.io/kubernetes/pkg/watch",
+		"k8s.io/kubernetes/pkg/client/unversioned",
+		"k8s.io/kubernetes/pkg/api/latest",
```

Comment:
Because I want the framework to automatically import the latest package as necessary. See here: https://github.com/kubernetes/kubernetes/pull/17699#discussion_r45954503.